README.md

```diff
@@ -73,27 +118,48 @@ or
 make install
 ```
 
-### Tests
-See the [contribution guide](CONTRIBUTING.md).
-
 # Configuration
 
-`lab` needs your GitLab information in order to interact with to your GitLab
-instance. There are several ways to provide this information to `lab`:
+_lab_ needs your GitLab information in order to interact with to your GitLab instance. There are several ways to
+provide this information to `lab`:
+
+### CI environment variables
 
-1. environment variables: `CI_PROJECT_URL`, `CI_JOB_TOKEN`, `GITLAB_USER_LOGIN`;
-    - Note: these are meant for when `lab` is running within a GitLab CI pipeline
-    - If all of these variables are set, the config files will not be updated.
-    - If all of these variables are set, these take precedence over all other values.
-2. environment variables: `LAB_CORE_HOST`, `LAB_CORE_TOKEN`;
-    - If these variables are set, the config files will not be updated.
-    - For example to use gitlab.com do: `export LAB_CORE_HOST="https://gitlab.com"`
-3. local configuration file in [Tom's Obvious, Minimal Language (TOML)](https://github.com/toml-lang/toml): `./lab.toml`;
-    - No other config files will be used as overrides if a local configuration file is specified
-4. user-specific configuration file in TOML: `~/.config/lab/lab.toml`.
-    - An example of user-specfic configurations in lab.toml can be found below.  As seen in the example file below, any command options specified by --help (for example 'lab mr show --help', or 'lab issue edit --help') can be set in the configuration file using TOML format.
+The environment variables `CI_PROJECT_URL`, `CI_JOB_TOKEN` and `GITLAB_USER_LOGIN`, intended to be used in a CI
+environment, can be set to prevent any configuration file creation/update. Also, any of these take precedence over all
+other options. 
 
```

Comment:
I've thought about this before and maybe it's time to remove this text?  It's only really useful in our CI environment and isn't something anyone else other than us is going to use.

Reply:
Not sure about that. I remember seeing someone in some issue mentioning they were using lab from within a CI env.

Reply:
Good point.  But I still think this section needs work.  Right now when I read this section I come away with the idea that CI_* variables are the feature I'm supposed to use.  Perhaps reorganizing the subsections here so that the README contains Fresh start, Configuration file, then local, and then the CI variables would be better for new users?

Reply:
Ok, it makes sense indeed. Let me reorder them.

Reply:
@prarit done. I slightly changed the _fresh start_ subsection.

Reply:
@prarit any feedback on this?